allinbits/cosmos-cash
# feat: add authentication data structure to did document

### Description

Added auth data structure to the did document

### How to test

- `make init-dev`
- `cd cmd/cosmos-cashd && go run main.go start`

In another terminal
- `go run main.go tx identifier create as --from validator --chain-id cash -y`
- `go run main.go query identifier identifiers --output json --node tcp://localhost:26657  | jq`
- See auth output
